recipes/mp-units/0.8.0/conanfile.py

```diff
@@ -0,0 +1,155 @@
+import os
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rmdir
+from conan.tools.scm import Version
+
+required_conan_version = ">=1.59.0"
+
+
+class MPUnitsConan(ConanFile):
+    name = "mp-units"
+    homepage = "https://github.com/mpusz/units"
+    description = "Physical Quantities and Units library for C++"
+    topics = (
+        "units",
+        "dimensions",
+        "quantities",
+        "dimensional-analysis",
+        "physical-quantities",
+        "physical-units",
+        "system-of-units",
+        "system-of-quantities",
+        "isq",
+        "si",
+        "library",
+        "quantity-manipulation",
+    )
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+    tool_requires = "cmake/3.26.4"
```

Comment:
If the library needs for specific CMake versions it would be better to use a Version range for CMake, there are some guidelines regarding Version Ranges here: https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#version-ranges

Reply:
It requires at least 3.19 which is not provided on at least one of your CI platforms.

Reply:
> I tried to set 3.20 but it failed to build with Conan V2. Which CMake version range is safe for V2?

Checking the CMake packages in conancenter I think it should work for CMake>=3.20.6

Reply:
Sure, I understand but as I said CMake 3.20 failed to download with Conan 2.0. I want to prevent that so I would prefer to narrow the range to versions that will work with Conan 2.0.

Reply:
I think It probably failed because you were not using version ranges, if using version ranges Conan will pick the versions that have packages for v2 so it will pick 3.19.8 or 3.20.6 that should work